frontend/src/metabase/dashboard/components/DashboardParameterList/DashboardParameterList.tsx

```diff
@@ -34,6 +39,7 @@ export function DashboardParameterList() {
       setEditingParameter={setEditingParameter}
       setParameterValueToDefault={setParameterValueToDefault}
       enableParameterRequiredBehavior
+      vertical={vertical}
```

Comment:
Dang, I searched on master, and we never used `vertical={true}` even before? And we're allowing our customers to use it first 😆